### PR TITLE
Add global nav and make it full width

### PR DESCRIPTION
**Pull request by deadlight**
## Done
Added the global nav

## QA
 - ./run
 - Go to any page
 - See that the global nav is present and full-width

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "lint-staged": "^7.2.2",
     "node-sass": "^4.5.3",
     "postcss-cli": "^4.1.0",
+    "prettier": "^1.15.2",
     "sass-lint": "^1.10.2",
     "vanilla-framework": "1.8.0",
     "watch-cli": "^0.2.2"
```
Comment (anthonydillon) [inline package.json:39]: I don't see the global nav dep: https://www.npmjs.com/package/global-nav

File: static/sass/styles.scss [hunk 2]
```diff
@@ -18,3 +18,9 @@
   max-width: none !important;
   width: 100%;
 }
+
+// Fix for full-width global nav
+.global-nav .global-nav__strip {
+  margin: 0 auto !important;
+  max-width: $grid-max-width;
```
Comment (anthonydillon) [inline static/sass/styles.scss:25]: This should be handled by the `maxWidth` option

File: package.json [hunk 3]
```diff
@@ -32,6 +32,7 @@
     "lint-staged": "^7.2.2",
     "node-sass": "^4.5.3",
     "postcss-cli": "^4.1.0",
+    "prettier": "^1.15.2",
```
Comment (anthonydillon) [inline package.json:36]: Did you mean to install prettier?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some suggestions in the code

File: package.json (hunk 3 above)
Comment (deadlight) [reply package.json:36]: Yep, it's used to check JS on commit

File: static/sass/styles.scss (hunk 2 above)
Comment (deadlight) [reply static/sass/styles.scss:25]: No, that's for the outer wrapper, the inner content needs to be constrained. Design & UX specified this. I'll submit a bug to the global-nav project to fix this upstream

**Comment by deadlight** [comment]
@anthonydillon ready for re-review

File: static/js/global-nav.js [hunk 4]
```diff
@@ -0,0 +1 @@
+../../node_modules/global-nav/dist/index.js
```
Comment (anthonydillon) [inline static/js/global-nav.js]: A better solution is to update the npm script that builds the site to `cp` this file instead of using a symlink

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Sorry, the way we using scripts from node_modules we cp the file in the build step.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Na, soz

**Comment by deadlight** [comment]
@anthonydillon @tbille how about now?!

**Comment by anthonydillon** [comment]
You didn't commit any changes

**Comment by anthonydillon** [comment]
You didn't commit any changes

**Comment by deadlight** [comment]
@anthonydillon whoops... it failed because I rewrote history. Forced.

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Comment by therealjuan-zz** [comment]
Any particular reason why the header is not fixed width as ubuntu.com? It looks strange to have a sub-domain with a completely different treatment to the header and things jumping around when you go from one side to the other one.

**Pull request merged**